.travis.yml

```diff
@@ -18,9 +18,10 @@ addons:
     - pkg-config gfortran netcdf-bin libnetcdf-dev libnetcdff-dev openmpi-bin libopenmpi-dev
 
 env:
-  - CPPFLAGS_ADD='' FCFLAGS_REAL_KIND='-fdefault-real-8 -fdefault-double-8' OPENMP=''
-  - CPPFLAGS_ADD='-DOVERLOAD_R4 -DOVERLOAD_R8' FCFLAGS_REAL_KIND='' OPENMP=''
-  - CPPFLAGS_ADD='' FCFLAGS_REAL_KIND='-fdefault-real-8 -fdefault-double-8' OPENMP='-fopenmp'
+  - FCFLAGS_ADD='' CONFIGURE_OPTIONS=''
+  - FCFLAGS_ADD='' CONFIGURE_OPTIONS='--without-openmp'
```

Comment:
The build for this compiles with -fopenmp.  It might be better to add -fno-openmp to the `FCFLAGS`

Reply:
The latest push in this PR resolves this.